homeassistant/components/bbox/sensor.py

```diff
@@ -134,7 +134,7 @@ def update(self):
         uptime = utcnow() - timedelta(
             seconds=self.bbox_data.router_infos["device"]["uptime"]
         )
-        self._attr_native_value = uptime.replace(microsecond=0).isoformat()
+        self._attr_native_value = uptime.replace(microsecond=0)
```

Comment:
Precision is handled by sensor itself, we don't need to set microsecond to 0

Reply:
https://github.com/home-assistant/core/pull/61219